### PR TITLE
mining: add section dedicated to p2pool

### PR DESCRIPTION
**Pull request by erciccione**
When the Moneropedia entry will be ready (#1845) we can also add links to that entry.

### CODE REVIEW

File: _i18n/en.yml [hunk 1]
```diff
@@ -559,6 +559,20 @@ mining:
   software_para: Note that some miners may have developer fees.
   support: Support
   supportp: If you have questions or just want to confront with fellow miners, come chat on Monero Pools. On <a href="#monero-pools:monero.social">Matrix</a> and <a href="irc://irc.libera.chat/#monero-pools">Libera</a>.
+  p2poolh: "P2Pool: The best of both solo and pool mining"
+  p2poolnew: >
+    P2Pool is a new clever new way of mining Monero, which allows to receive frequent payouts like if you are mining in a pool, but while mining solo. It's a Peer-To-Peer mining pool that gives you full control over your Monero node and what it mines. More details in <a href="{{ site.baseurl }}/2021/10/05/p2pool-released.html">the announcement post.
```
Comment (selsta) [inline _i18n/en.yml:564]:
```suggestion
    P2Pool is a new clever way of mining Monero, which allows to receive frequent payouts like if you are mining in a pool, but while mining solo. It's a Peer-To-Peer mining pool that gives you full control over your Monero node and what it mines. More details in <a href="{{ site.baseurl }}/2021/10/05/p2pool-released.html">the announcement post.
```

Comment (erciccione) [reply _i18n/en.yml]: Fixed. Thanks.

File: _i18n/en.yml [hunk 2]
```diff
@@ -559,6 +559,20 @@ mining:
...
+    P2Pool is a sidechain to Monero, and P2Pool blocks are potentially Monero blocks. Each miner submits block templates that include a payout for all of the miners that currently have shares in the PPLNS window. High quality block templates are added to the P2Pool blockchain as blocks which count as "shares" for the miner who found them.
+  p2poolfeatures: Main features
+  p2pdecentralized: <b>Decentralized:</b> no central server that can be shutdown/blocked.
+  p2ppermissionless: <b>Permissionless:</b> there is no one to decide who can mine on the pool and who can't.
+  p2ptrustless: <b>Trustless:</b> there is no pool wallet; funds are never in custody. All pool blocks pay out to miners immediately.
+  p2pplns: <b>PPLNS</b> payout scheme
+  p2pzerofee: 0% fee
+  p2pzeropayout: <b>0 XMR</b> payout fee
+  p2pminpayout: Less than 0.0004 XMR minimum payout
+  p2pmoreinfo: "More information and details on the GitBub repository of the project:"
```
Comment (selsta) [inline _i18n/en.yml:575]:
```suggestion
  p2pmoreinfo: "More information and details on the GitHub repository of the project:"
```

File: _i18n/en.yml [hunk 3]
```diff
@@ -559,6 +559,20 @@ mining:
   software_para: Note that some miners may have developer fees.
   support: Support
   supportp: If you have questions or just want to confront with fellow miners, come chat on Monero Pools. On <a href="#monero-pools:monero.social">Matrix</a> and <a href="irc://irc.libera.chat/#monero-pools">Libera</a>.
+  p2poolh: "P2Pool: The best of both solo and pool mining"
+  p2poolnew: >
+    P2Pool is a new clever way of mining Monero, which allows to receive frequent payouts like if you are mining in a pool, but while mining solo. It's a Peer-To-Peer mining pool that gives you full control over your Monero node and what it mines. More details in <a href="{{ site.baseurl }}/2021/10/05/p2pool-released.html">the announcement post.
```
Comment (SamsungGalaxyPlayer) [inline _i18n/en.yml:564]: P2Pool is a clever new way of mining Monero, which allows miners to receive the frequent payouts offered by pools without needing to trust a centralized pool. P2Pool is a Peer-To-Peer mining pool that gives miners full control over their Monero node and what it mines. More details in <a href="{{ site.baseurl }}/2021/10/05/p2pool-released.html">the announcement post.

File: _i18n/en.yml [hunk 4]
```diff
@@ -559,6 +559,20 @@ mining:
...
+  p2poolp: >
+    P2Pool is a sidechain to Monero, and P2Pool blocks are potentially Monero blocks. Each miner submits block templates that include a payout for all of the miners that currently have shares in the PPLNS window. High quality block templates are added to the P2Pool blockchain as blocks which count as "shares" for the miner who found them.
```
Comment (SamsungGalaxyPlayer) [inline _i18n/en.yml:566]: P2Pool is a sidechain to Monero, and P2Pool blocks are potentially Monero blocks. Each miner submits block templates that include payouts to all of the miners that are mining at the same time (those that currently have shares in the PPLNS window). High quality block templates are added to the P2Pool blockchain as blocks; these count as "shares" for the miner who found them.

File: _i18n/en.yml [hunk 5]
```diff
@@ -559,6 +559,20 @@ mining:
...
+  p2poolfeatures: Main features
+  p2pdecentralized: <b>Decentralized:</b> no central server that can be shutdown/blocked.
+  p2ppermissionless: <b>Permissionless:</b> there is no one to decide who can mine on the pool and who can't.
```
Comment (SamsungGalaxyPlayer) [inline _i18n/en.yml:569]: Permissionless: anyone can join the sidechain, and it's effectively impossible to censor individual miners.

File: _i18n/en.yml [hunk 6]
```diff
@@ -559,6 +559,20 @@ mining:
...
   support: Support
   supportp: If you have questions or just want to confront with fellow miners, come chat on Monero Pools. On <a href="#monero-pools:monero.social">Matrix</a> and <a href="irc://irc.libera.chat/#monero-pools">Libera</a>.
+  p2poolh: "P2Pool: The best of both solo and pool mining"
+  p2poolnew: >
+    P2Pool is a new clever way of mining Monero, which allows to receive frequent payouts like if you are mining in a pool, but while mining solo. It's a Peer-To-Peer mining pool that gives you full control over your Monero node and what it mines. More details in <a href="{{ site.baseurl }}/2021/10/05/p2pool-released.html">the announcement post.
+  p2poolp: >
+    P2Pool is a sidechain to Monero, and P2Pool blocks are potentially Monero blocks. Each miner submits block templates that include a payout for all of the miners that currently have shares in the PPLNS window. High quality block templates are added to the P2Pool blockchain as blocks which count as "shares" for the miner who found them.
+  p2poolfeatures: Main features
+  p2pdecentralized: <b>Decentralized:</b> no central server that can be shutdown/blocked.
```
Comment (SamsungGalaxyPlayer) [inline _i18n/en.yml:568]: Decentralized: no central server that can be shutdown/blocked/malicious.

**Review by SamsungGalaxyPlayer — COMMENTED** [review]
Suggestions for easier readability. This is a good thing to have on the website.

**Review by selsta — APPROVED** [review]

**Pull request merged**